src/components/Navbar.js

```diff
@@ -12,13 +12,13 @@ class NavBar extends React.Component {
         </div>
       </div>
     );
-    
+
     const userNav = (
       <div className="ui stackable teal pointing menu">
-        <NavLink className="item" activeClassName="item active" exact to="/"><i className="fa fa-free-code-camp" />freeCodeCamp Alumni Network</NavLink>
-        <NavLink className="item" activeClassName="item active" exact to="/dashboard">Dashboard</NavLink>
-        <NavLink className="item" activeClassName="item active" exact to="/">Profile</NavLink>
-        <NavLink className="item" activeClassName="item active" exact to="/">Interests</NavLink>
+        <NavLink className="item" activeClassName="active item" exact to="/"><i className="fa fa-free-code-camp"></i>freeCodeCamp Alumni Network</NavLink>
```

Comment:
Minor, but I don't think the `<i>` needs to have a separate closing tag here, since no children.